freqtrade/strategy/interface.py

```diff
@@ -558,12 +567,12 @@ def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool,
                     sell: bool, low: float = None, high: float = None,
                     force_stoploss: float = 0) -> SellCheckTuple:
         """
-        This function evaluates if one of the conditions required to trigger a sell
-        has been reached, which can either be a stop-loss, ROI or sell-signal.
-        :param low: Only used during backtesting to simulate stoploss
-        :param high: Only used during backtesting, to simulate ROI
+        This function evaluates if one of the conditions required to trigger a sell/exit_short
+        has been reached, which can either be a stop-loss, ROI or exit-signal.
+        :param low: Only used during backtesting to simulate (long)stoploss/(short)ROI
```

Comment:
This comment is wrong. Also on shorts it's called stoploss (it's on the negative side of the trade), not ROI.

Reply:
What part about it is wrong? On shorts, the roi would be a low value, and the stoploss would be a high value